### PR TITLE
Reviewscreen

### PR DESCRIPTION
**Pull request by balogun14**

### CODE REVIEW

File: lib/apps/customer_app/features/review/ui/screens/review.screen.dart [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+class ReviewsScreen extends StatefulWidget {
+  const ReviewsScreen({super.key});
+
+  @override
+  State<ReviewsScreen> createState() => _ReviewsScreenState();
+}
+
+class _ReviewsScreenState extends State<ReviewsScreen> {
+  int rating = 0;
+  // This widget is the root of your application.
```
Comment (dro-1) [inline lib/apps/customer_app/features/review/ui/screens/review.screen.dart:15]: What did you mean by this comment?

File: lib/apps/customer_app/features/review/ui/screens/review.screen.dart [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
```
Comment (dro-1) [inline lib/apps/customer_app/features/review/ui/screens/review.screen.dart:18]: Try removing the MaterialApp Wrapper as it causes it to lose most of the theming that has already been set up.

File: lib/apps/customer_app/features/review/ui/screens/review.screen.dart [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                  child: Icon(
+                    Icons.close_outlined,
+                    size: 32.0,
+                  ),
+                ),
+                const SizedBox(
+                  height: 12,
+                ),
+                const ListWidget(
+                  text: 'Service fee',
```
Comment (dro-1) [inline lib/apps/customer_app/features/review/ui/screens/review.screen.dart:46]: The font of this text doesn't seem to align with what is in the UI.

File: lib/apps/customer_app/features/review/ui/screens/review.screen.dart [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+                    fontWeight: FontWeight.w700,
+                    height: 0,
+                    letterSpacing: 0.36,
+                  ),
+                ),
+                const Padding(
+                  padding: EdgeInsets.only(left: 28.0),
+                  child: SizedBox(
+                    height: 42,
+                    child: DottedLines(),
```
Comment (dro-1) [inline lib/apps/customer_app/features/review/ui/screens/review.screen.dart:61]: The DottedLines as well don't really look like the design. Try passing params to the widget to customise how it shows up.

File: lib/apps/customer_app/features/review/ui/screens/review.screen.dart [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+                  ),
+                ),
+                const ListWidget(
+                  text: 'Time Spent',
```
Comment (dro-1) [inline lib/apps/customer_app/features/review/ui/screens/review.screen.dart:65]: Most of the text doesn't really have the same font as the design

File: lib/apps/customer_app/features/review/ui/screens/review.screen.dart [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+                            rating = count;
+                          });
+                        },
+                      ),
+                    ],
+                  ),
+                ),
+              ],
+            ),
+          ),
```
Comment (dro-1) [inline lib/apps/customer_app/features/review/ui/screens/review.screen.dart:127]: The pill shaped boxes showing "Poor Repairs", "Arrived Late", etc are not showing.

File: lib/apps/customer_app/features/review/ui/screens/review.screen.dart [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+          floatingActionButton: Container(
+            width: 366.0,
+            height: 72.0,
+            margin: const EdgeInsets.all(20.0),
+            decoration: const BoxDecoration(
+                color: Color(0xFF14161C),
+                borderRadius: BorderRadius.all(Radius.circular(20))),
+            child: TextButton(
```
Comment (dro-1) [inline lib/apps/customer_app/features/review/ui/screens/review.screen.dart:135]: This "Done" button is not centered on the page

File: lib/apps/customer_app/features/review/ui/screens/review.screen.dart [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+              mainAxisAlignment: MainAxisAlignment.start,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                const SizedBox(
+                  height: 20,
+                ),
+                const Padding(
+                  padding: EdgeInsets.only(left: 12.0, bottom: 25),
+                  child: Icon(
+                    Icons.close_outlined,
```
Comment (dro-1) [inline lib/apps/customer_app/features/review/ui/screens/review.screen.dart:38]: The close button seems to be a tad bit higher up the screen than it is in the design.

**Review by dro-1 — COMMENTED** [review]
The naming conventions of the file and their folder structure are also not correct. 
I also can't tell what the EndHandees Widget and the _dialogBuilder function are being used for.
We should probably just have a call to trash everything out.

File: lib/apps/customer_app/features/review/ui/screens/review.screen.dart (hunk 8 above)
Comment (balogun14) [reply lib/apps/customer_app/features/review/ui/screens/review.screen.dart:38]: I'll adjust that

**Comment by balogun14** [comment]
@dro-1  regarding the theming you know i told you i was not able to run the app on my machine so i had to create a new Flutter project to test it that's why there is a problem with the theming. @dro-1  can we have a call today

**Pull request closed without merging**